src/components/Suggestions.js

```diff
@@ -102,6 +102,7 @@ class Suggestions extends Component {
           <li
             key={i}
             onMouseDown={props.handleClick.bind(null, i)}
+            onTouchStart={props.handleClick.bind(null, i)}
```

Comment:
better to bind this here so that event is passed.

Reply:
also can add spec for this file in suggestion.spec.js file to check if `handleclick` was called when touch start event is simulated

Reply:
handleClick is already bind

Reply:
there is not "suggestion.spec.js" file in the directory
Added the simulation of touchstart in suggestion.test.js. though there is no need to add it. since neither click nor hover is been simulated on suggestion.test.js

Reply:
@gagan876574 i meant check if handleClick prop is triggered when touchStart event is simulated on the  suggestion

Reply:
@gagan876574 you still haven't checked if handleClick prop is triggered when touchStart event is simulated on the suggestion

Reply:
Check now